backend/stylists/app/Http/Controllers/ProductController.php

```diff
@@ -144,14 +144,14 @@ public function getCreate(Request $request)
             return;
         }
 
-        $merchant = Merchant::where('name', $request->input('merchant'))->first();
-        $brand = Brand::where(['name' => $request->input('brand')])->first();
+        $merchant = Merchant::firstOrCreate(['name' => $request->input('merchant')]);
```

Comment:
Dont create merchants on fly. If merchant doesnt exist, error should be returned.